packages/libs/eda/src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -651,6 +652,18 @@ function HistogramViz(props: VisualizationProps<Options>) {
     ])
   );
 
+  // controls need the bin info from just one facet (not an empty one)
+  const checkData = isFaceted(data.value)
```

Comment:
Need to get rid of `data0` below and replace it with `checkData`

Reply:
oh I see. will do that

Reply:
done!